config/SGGame/pokemon.json

```diff
@@ -1,4 +1,25 @@
 {
+	"missingno": {
+		"expType": null,
+		"num": 0,
+		"rate": null,
```

Comment:
Rate should be a number to prevent errors, use 1 (or 0 if you can confirm the rate calc formula wont divide by 0)